src/sentry/sentry_metrics/indexer/postgres/postgres_v2.py

```diff
@@ -212,3 +219,9 @@ def reverse_shared_org_resolve(self, id: int) -> Optional[str]:
 class PostgresIndexer(StaticStringIndexer):
     def __init__(self) -> None:
         super().__init__(CachingIndexer(indexer_cache, PGStringIndexerV2()))
+        cleanup.EXTRA_BULK_QUERY_DELETES.extend(
+            [
+                (StringIndexerPostgresModel, "last_seen", None),
+                (PerfStringIndexerPostgresModel, "last_seen", None),
+            ]
+        )
```

Comment:
This does not work. Where did you plan to instantiate the `PostgresIndexer` for the changes to `EXTRA_BULK_QUERY_DELETES` to take effect?

```
❯ git rev-parse --abbrev-ref HEAD
john/metrics_indexer_cleanup

❯ git --no-pager diff
diff --git a/src/sentry/runner/commands/cleanup.py b/src/sentry/runner/commands/cleanup.py
index 5e1c180883..b7273476e2 100644
--- a/src/sentry/runner/commands/cleanup.py
+++ b/src/sentry/runner/commands/cleanup.py
@@ -174,6 +174,10 @@ def cleanup(days, project, concurrency, silent, model, router, timed):
                 return False
             return model.__name__.lower() not in model_list

+
+        print(f"{EXTRA_BULK_QUERY_DELETES=}")
+        exit(1)
+
         # Deletions that use `BulkDeleteQuery` (and don't need to worry about child relations)
         # (model, datetime_field, order_by)
         BULK_QUERY_DELETES = [

❯ sentry cleanup
{"name":"sentry.utils.geo","event":"settings.GEOIP_PATH_MMDB not configured.","level":"warning"}
{"name":"sentry.plugins.github","event":"apps-not-configured","level":"info"}
EXTRA_BULK_QUERY_DELETES=[]
```

I understand that we might not use the `PostgresIndexer` but since the models are present always, worst case, we query empty tables, so I suggest adding the models to the `BULK_QUERY_DELETES`.

Reply:
> This does not work. Where did you plan to instantiate the PostgresIndexer for the changes to `EXTRA_BULK_QUERY_DELETES` to take effect?
I was plagiarizing from [6327](https://github.com/getsentry/sentry/pull/6327/files#diff-7173069809318e0fe8909c21eac05d4cef36e20cd1e14d7833d3bd135bd95625R64).
From @wedamija's comment I was expecting `PostgresIndexer` to be [loaded](https://github.com/getsentry/sentry/blob/00845f9617523b5a98f07b2457696f6ebca8a33b/src/sentry/sentry_metrics/consumers/indexer/processing.py#L24) and [instantiated](https://github.com/getsentry/sentry/blob/00845f9617523b5a98f07b2457696f6ebca8a33b/src/sentry/sentry_metrics/consumers/indexer/processing.py#L31) in `processing.py`, which will update `EXTRA_BULK_QUERY_DELETES`. But I could be wrong because this is my first time working with the sentry codebase :)

Reply:
Might be easier to just assume it exists and go back to how we implemented it earlier, since I guess there's a chance that the code doesn't end up imported

Reply:
Sounds good, reverting to original